### ISSUE TITLE
question about the decrease process of the loss

### ISSUE THREAD

**Issue by ofishhaha** [dichotomies/proxy-nca#8]
I noticed that during the training process, your proxy loss only decreased a little. Although the result is similar with the result in paper, maybe it will further increase if the loss keep decreasing. Do you have any idea why this happens?

I tried to replace the loss by using traditional  triplet loss. Also the loss is difficult to decrease. And the reproduced rank-1 accuracy is only 34%. Would you please tell me what should i change?

Thank you so much for your effort.

**Comment by dichotomies** [comment]
The model starts to overfit at some point. Hence, reducing the loss further wouldn't be beneficial. But you can see yourself: just set `--gamma` to 1, this will result in a non-decreasing learning rate and thus, higher decrease of the loss.

You say that you only achieve R@1 of 34% for this setup (Proxy NCA with default arguments)?

**Comment by dichotomies** [comment]
You might want to check the updated version. The loss is not affected from this problem any longer.
